docker/testnet/testnet-container.env

```diff
@@ -0,0 +1,22 @@
+## These variables should be defined by the user before configuring/running a node 
+
+## MAINNET VERSION
```

Comment:
Should be testnet

Reply:
I removed both, the network is clearly defined in line 14.